Client/Ecosia/UI/NTP/NTPTooltip.swift

```diff
@@ -98,6 +133,26 @@ final class NTPTooltip: UICollectionReusableView, NotificationThemeable {
         accessibilityTraits = .button
         isAccessibilityElement = true
     }
+    
+    func removeLink() {
+        linkButton.setTitle(nil, for: .normal)
+        linkButton.isHidden = true
+    }
+    
+    func setLink(_ text: String) {
```

Comment:
Make sense to have a string as we are operating with an attributed one within the function. Wonder whether it would sound more consistent to pass a URL rather than a `text: String` 💭 . WDYT?